src/Symfony/Component/Security/Core/Exception/LazyResponseException.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Security\Core\Exception;
+
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Wraps a lazily computed response in a signaling exception.
```

Comment:
`Signaling exception that wraps a lazily computed response` maybe? I'm wondering because `LazyResponseEvent` has the exact same docblock

Reply:
updated